MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -10,46 +11,58 @@
 namespace MauiAppBasic.MSALClient
 {
     /// <summary>
-    /// This is a wrapper for PCA. It is singleton and can be utilized by both application and the MAM callback
+    /// This is a wrapper for PublicClientApplication. It is singleton and can be utilized by both application and the MAM callback
     /// </summary>
     public class PCAWrapper
     {
         /// <summary>
-        /// This is the singleton used by consumers
+        /// This is the singleton used by consumers. Since PCAWrapper constructor does not have perf or memory issue, it is instantiated directly.
         /// </summary>
         public static PCAWrapper Instance { get; private set; } = new PCAWrapper();
 
+        /// <summary>
+        /// Instance of PublicClientApplication. It is provided, if App wants more customization.
+        /// </summary>
         internal IPublicClientApplication PCA { get; }
 
-        internal bool UseEmbedded { get; set; } = false;
-
         /// <summary>
-        /// The authority for the MSAL PublicClientApplication. Sign in will use this URL.
+        /// This will determine if the Interactive Authentication should be Embedded or System view
         /// </summary>
-        private const string _authority = "https://login.microsoftonline.com/common";
+        internal bool UseEmbedded { get; set; } = false;

```

Comment:
I think we should not use `internal` in samples. Let's just use public and private.

Reply:
Internal is used to reduce the attack surface. If it is used outside the class and but just withing the assembly, IMO, internal should be used. I would minimize the use of public. That is my reasoning behind it.